salt/orch/kubernetes.sls

```diff
@@ -3,6 +3,13 @@ mine_update:
     - name: mine.update
     - tgt: '*'
 
+sync_modules:
+  salt.function:
+    - name: saltutil.sync_modules
```

Comment:
Yea, `saltutil.sync_modules` pushes the modules out to the minions's `/var/cache/salt/minion/extmods/` dir where they can be loaded by the minion (and the `refresh=True` forces an an immediate reload of extension modules)